release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1883,16 +1883,24 @@ As a workaround, if you have an installation configuration file, update it with
 
-* The following known issue applies to users who updated their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4: 
+* The following known issue applies to users who updated their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4:
++
+After the introduction of the node identify feature, some pods that were running as root are updated to run unprivileged. For users who updated to an early access version of {product-title} {product-version}, attempting to upgrade to the official version of {product-version} might not progress. In this scenario, the Network Operator reports the following state, indicating an issue with the update: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`.
 +
-After the introduction of the node identify feature, some pods that were running as root are updated to run unprivileged. For users who updated to an early access version of {product-title} {product-version}, attempting to upgrade to the official version of {product-version} might not progress. In this scenario, the Network Operator reports the following state, indicating an issue with the update: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`. 
-+ 
 As a workaround, you can delete all pods in the `openshift-network-node-identify` namespace by running the following command: `oc delete --force=true -n openshift-network-node-identity --all pods`. After running this command, the update continues.
 +
-For more information about early access, xref:../updating/understanding_updates/understanding-update-channels-release.adoc#candidate-version-channel_understanding-update-channels-releases[candidate-4.14 channel]. 
+For more information about early access, xref:../updating/understanding_updates/understanding-update-channels-release.adoc#candidate-version-channel_understanding-update-channels-releases[candidate-4.14 channel].
 
 * Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a Daemon Set, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
 
+* There is a known issue with using Azure confidential virtual machines, which is a Technology Preview feature in this release. Configuring a cluster to encrypt the managed disk and the Azure VM Guest State (VMGS) blob with a platform-managed key (PMK) or a customer-managed key (CMK) is unsupported. To avoid this issue, only enable encryption of the VMGS blob by setting the value of the `securityEncryptionType` parameter to `VMGuestStateOnly`. (link:https://issues.redhat.com/browse/OCPBUGS-18379[*OCPBUGS-18379*])
```

Comment:
Should we even document `DiskWithVMGuestState`? I am thinking it might make sense to remove that param from docs until it is supported.

Currently exists in #65764

Reply:
I agree, that we should probably skip the documention for  `DiskWithVMGuestState`, but I would still include this change here. This is what you're suggesting @jeana-redhat, right?

Reply:
Yes, I think leave the release note in case anyone digs around and finds ` DiskWithVMGuestState`, but remove ` DiskWithVMGuestState` from the usage docs.

Cc: @bscott-rh for the install side of that